doc/adb.md

```diff
@@ -1,262 +0,0 @@
-# SE/30 Startup Sequence
```

Comment:
I agree with cleaning up the documentation, but I dont' think it should be part of this PR. We can work through that as part of a different PR. Please restore these files, as well as the case and hw directories.

Additionally, please restore the hardware and case directories. We can have a discussion about moving these to a different location but that discussion should occur outside this PR.